src/Faker/Provider/en_CA/PhoneNumber.php

```diff
@@ -2,7 +2,7 @@
 
 namespace Faker\Provider\en_CA;
 
-class PhoneNumber extends \Faker\Provider\PhoneNumber
+class PhoneNumber extends \Faker\Provider\en_CA\PhoneNumber
```

Comment:
I think there is a typo in your proposed commit (causing it to fail CI). Did you mean?

```suggestion
class PhoneNumber extends \Faker\Provider\en_US\PhoneNumber
```